tests/php/sync/test_class.jetpack-sync-full.php

```diff
@@ -1615,6 +1615,36 @@ function test_full_sync_sends_previous_interval_end_on_comments() {
 		$this->full_sync->reset_data();
 	}
 
+	function test_disable_sending_full_sync() {
+		$this->factory->post->create_many( 2 );
+
+		$this->sender->reset_data();
+		$this->server_event_storage->reset();
+
+		Settings::update_settings( array( 'send_enabled_full_sync' => 0 ) );
+
+		$this->full_sync->start();
+		$this->sender->do_full_sync();
+
+		$start_event = $this->server_event_storage->get_most_recent_event( 'jetpack_full_sync_start' );
+		$this->assertTrue( ! $start_event );
+	}
+
+	function test_enable_sending_full_sync() {
+		$this->factory->post->create_many( 2 );
+
+		$this->sender->reset_data();
+		$this->server_event_storage->reset();
+
+		Settings::update_settings( array( 'send_enabled_full_sync' => 1 ) );
+
+		$this->full_sync->start();
+		$this->sender->do_full_sync();
+
+		$start_event = $this->server_event_storage->get_most_recent_event( 'jetpack_full_sync_start' );
+		$this->assertTrue( $start_event );
```

Comment:
$start_event is an object not just `true`
```suggestion
		$this->assertTrue( ! empty( $start_event ) );
```